optuna_dashboard/_storage_url.py

```diff
@@ -98,15 +98,15 @@ def get_journal_file_storage(file_path: str) -> JournalStorage:
         raise ValueError("JournalRedisStorage is available from Optuna v3.1.0")
 
     from optuna.storages import JournalFileOpenLock
-    from optuna.storages import JournalFileStorage
+    from optuna.storages import JournalFileBackend
```

Comment:
`JournalFileBackend` is supposed to be imported from `optuna.storages.journal`.
See https://optuna.readthedocs.io/en/stable/reference/generated/optuna.storages.journal.JournalFileBackend.html